### DISCUSSION TITLE
Help installing using BRAT

### DISCUSSION THREAD

**Discussion by spark706** [TfTHacker/obsidian42-strange-new-worlds#23] [Q&A] [answered]
I have tried installing SNW using BRAT using the Github repo title but every time it tells me there is no manifest.json file. I even tried just enabling the plugin in the folder and it says the same thing.  What am I doing wrong?

**Comment by TfTHacker** [comment, marked as answer]
Sorry about that, there was a problem with the packaging. Please try it again. Also just make sure when you use the "add plugin" feature in BRAT is to use the path: https://github.com/TfTHacker/obsidian42-strange-new-worlds

**Reply by spark706** [reply]
That's the trick. Thanks.  BTW, with the example entry in BRAT, it simply shows the repo owner and the particular repo.  Would it be possible to add the whole link in the example or is that an implied thing?

**Comment by TfTHacker** [comment]
Tnx for getting back to me. Yes the faint link path is just to give an idea of what it should look like. You think it should be different?

**Reply by spark706** [reply]
My 2 cents, maybe just add the https portion in the faint writing.  I tend to see those sorts of things as what is expected.  For me, I started with the whole path and when I got the error, I relooked and saw just TfTHAcker/.... and did that.

**Reply by TfTHacker** [reply]
Good idea. Will push out an update to BRAT with this.

**Comment by spark706** [comment]
I sometimes get good ideas.  Cheers!
